cfme/base/__init__.py

```diff
@@ -105,6 +105,29 @@ def update_advanced_settings(self, settings_dict):
         )
         assert result.ok
 
+    def upload_custom_logo(self, file_type, file_data=None, enable=True):
+        """
+        This function can be used to upload custom logo or text and use them.
+
+        Args:
+            file_type (str) : Can be either of [logo, login_logo, brand, favicon, logintext]
+            file_data (str) : Text data if file_type is logintext else image name to be uploaded
+            enable (bool) : True to use the custom logo/text else False
+        """
+        view = navigate_to(self, "CustomLogos")
+        logo_view = getattr(view.customlogos, file_type)
```

Comment:
I would check attribute collected or not else raise `ValueError` means if `file_type` not matched.